app/labor/cache_buster.rb

```diff
@@ -111,4 +111,62 @@ def bust_tag_pages(article)
       end
     end
   end
+
+  def bust_page(slug)
+    bust "/page/#{slug}"
+    bust "/page/#{slug}?i=i"
+    bust "/#{slug}"
+    bust "/#{slug}?i=i"
+  end
+
+  def bust_tag(name)
+    bust("/t/#{name}")
+    bust("/t/#{name}?i=i")
+    bust("/t/#{name}/?i=i")
+    bust("/t/#{name}/")
+    bust("/tags")
+  end
+
+  def bust_event
+    bust("/events")
+    bust("/events?i=i")
+  end
+
+  def bust_podcast(path)
+    bust("/" + path)
+  end
+
+  def bust_organization(organization, slug)
+    bust("/#{slug}")
+    begin
+      organization.articles.find_each do |article|
+        bust(article.path)
+      end
+    rescue StandardError => e
+      Rails.logger.error("Tag issue: #{e}")
+    end
+  end
+
+  def bust_podcast_episode(podcast_episode, path, podcast_slug)
+    podcast_episode.purge
+    podcast_episode.purge_all
+    begin
```

Comment:
Strange busting: busting the same url multiple times and rescuing `StandardError`, but that's not a topic of this pr.